# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../murrine-configurator/themedir.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./files/murrine-configurator.orig	2006-09-17 02:45:58.000000000 +0300
+++ ./files/murrine-configurator	2006-09-19 14:42:25.000000000 +0300
@@ -50,7 +50,7 @@
 
 function select_theme {
 	if [ "$PREFIX" = "" ]; then
-		PREFIX=~/.themes
+		PREFIX=/usr/share/themes
   fi
 	THEMES=`ls ${PREFIX} | grep Murrina | sed s@~/${PREFIX}/@@g`
 
